main/main.cpp

```diff
@@ -1826,6 +1826,8 @@ Error Main::setup(const char *execpath, int argc, char *argv[], bool p_second_ph
 	Engine::get_singleton()->set_physics_jitter_fix(GLOBAL_DEF("physics/common/physics_jitter_fix", 0.5));
 	Engine::get_singleton()->set_max_fps(GLOBAL_DEF(PropertyInfo(Variant::INT, "application/run/max_fps", PROPERTY_HINT_RANGE, "0,1000,1"), 0));
 
+	GLOBAL_DEF("debug/rendering/force_wireframe", -1.0f);
```

Comment:
Shouldn't the default value be `0.0`?
Also, a property hint should be added so that you can't set nonsensical values (such as values below `0.0`, or values above `128.0` or so).

Reply:
> Shouldn't the default value be `0.0`? 

That's just my superstitions towards floating point comparisons speaking, 0.0 is probably the more sensible default.